lib/matplotlib/streamplot.py

```diff
@@ -58,6 +59,10 @@ def streamplot(axes, x, y, u, v, density=1, linewidth=None, color=None,
         In data coordinates, the same as the ``x`` and ``y`` arrays.
     *zorder* : int
         any number
+    *maxlength* : float
+        Maximum length of streamline in axes coordinates.
+    *integration_direction* : ['forward', 'backward', 'both']
```

Comment:
Oh, according to numpydoc, it should be braces `{}` instead of brackets `[]`, then the first value is the default.